### PR TITLE
Add vscode-openshift-connector 0.1.4 image

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2019 Red Hat, Inc.
```
Comment (benoitf) [inline Dockerfile]:
```suggestion
# Copyright (c) 2020 Red Hat, Inc.
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+FROM alpine:3.10.2
```
Comment (benoitf) [inline Dockerfile]: probably needs to bump to the latest version of alpine as well `3.11.3` or or `3.10.4` ?

File: etc/entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#
+# Copyright (c) 2018-2018 Red Hat, Inc.
```
Comment (benoitf) [inline etc/entrypoint.sh]:
```suggestion
# Copyright (c) 2020 Red Hat, Inc.
```

**Comment by ericwill** [comment]
Can we incorporate the changes from [Nick's PR](https://github.com/che-dockerfiles/che-sidecar-openshift-connector/pull/7) here as well? For consistency's sake.

**Pull request closed without merging**